index.html

```diff
@@ -24,13 +24,11 @@
             <h1>Hey There.
                 I’m Michael</h1>
         </section>
-        <section class="intro-title">
-            <h3>I’am a Software Developer</h3>
-        </section>
-        <section class="intro-detail">
```

Comment:
The whole Headline should be wrapped inside a `section` container. You may use section tags for each section inside a section container. Moreover, please add background images(If in the design) to this section.